### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by MasterSINISTER**
Added my First Open Source Contribution

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -111,11 +111,38 @@ <h2>A project for first-time contributions</h2>
...
+          <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
+          <i class="fab fa-github"></i>
+          <a href="https://www.github.com/MasterSINISTER" target="_blank">Github</a>
+        </p>
+        <p class="about">Full-stack developer and technical leader is driven by a passion for problem-solving and continuous learning.</p>
+        <div class="resources">
+          <p>3 Useful Dev Resources</p>
+          <ul>
+            <li>
+              <a href="#" target="_blank" title="First Resource">GeeksForGeeks</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:134]: Please add a valid href attribute and an accessible title.

File: index.html [hunk 2]
```diff
@@ -111,11 +111,38 @@ <h2>A project for first-time contributions</h2>
...
+            </li>
+            <li>
+              <a href="#" target="_blank" title="Second Resource">Daily.dev</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:137]: Please add a valid href attribute and an accessible title.

File: index.html [hunk 3]
```diff
@@ -111,11 +111,38 @@ <h2>A project for first-time contributions</h2>
...
+            </li>
+            <li>
+              <a href="#" target="_blank" title="Third resource">Medium</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:140]: Please add a valid href attribute and an accessible title.

**Comment by CandidDeer** [comment]
@MasterSINISTER Your card is still missing the href data, that is your resources' links. Can you please add them? Thx.

**Comment by MasterSINISTER** [comment]
Fixed It

**Pull request closed without merging**